cmd/check_operator.go

```diff
@@ -13,7 +13,7 @@ import (
 
 var checkOperatorCmd = &cobra.Command{
 	Use:   "operator",
-	Short: "Run checks for an Operator",
+	Short: "Run checks for an operator",
```

Comment:
Hehe....shouldn't "Operator" be upper case?  Operator is a proper noun no?

Reply:
Haha! I just wanted it to be consistent, and container is lower case.

That said... I Tend To Capitalize Way Too Much and so that's why it caught my eye 😆 😆

Reply:
I think Operator is actually capitalized and container is lower case. I looked at how they do it on RedHat.com and that seems to be their implementation.

Reply:
Fixed.